.github/CODEOWNERS

```diff
@@ -10,10 +10,13 @@
 # The Carbon QA team are responsible for reviewing Pull Requests
 # that contain any change modifying spec and test.js files 
```

Comment:
Should this comment be amended to include tsx and Playwright references? Something like the following, perhaps? "The Carbon QA team are responsible for reviewing Pull Requests that contain any change modifying spec and cy.tsx or pw.tsx files, or anything in the Cypress or Playwright directories."